server/src/resolvers/mutations/createChild.js

```diff
@@ -0,0 +1,25 @@
+const { getUserId } = require('../../utils')
+
+module.exports = {
+    createChild: async (_, { firstName, lastName, rateAmount, rateType, gender }, context) => {
+        const userId = getUserId(context)
+
+        // TODO: Handle creating children for owners who aren't created yet --- is this possible?
```

Comment:
Yes it is 

In the create child we can accept a create instead of connect on `owner`, We can discuss this later!